### PR TITLE
[SCAL-90574]-Added coverageThreshold for /src folder

### PR DESCRIPTION
**Pull request by harsh-ts**
ticket id https://thoughtspot.atlassian.net/browse/SCAL-90574

### CODE REVIEW

**Review by ashubham — APPROVED** [review]

File: jest.config.sdk.js [hunk 1]
```diff
@@ -6,6 +6,14 @@ module.exports = {
     coverageDirectory: 'coverage/sdk/',
     coverageReporters: ['lcov', 'text', 'cobertura'],
     coveragePathIgnorePatterns: ['/node_modules/', '/test/'],
+    coverageThreshold: {
+        './src/': {
+            branches: 80,
+            functions: 80,
+            lines: 80,
+            statements: -10,
```
Comment (ashubham) [inline jest.config.sdk.js:14]: whats this -10 for ?

Comment (harsh-ts) [reply jest.config.sdk.js:14]: if there are more than 10 uncovered statements

File: src/embed/base.ts [hunk 2]
```diff
@@ -241,6 +245,18 @@ export class TsEmbed {
         return event.data?.type || event.data?.__type;
     }
 
+    /**
+     * Extracts the port field from the event payload
+     * @param event  The window message event
+     * @returns
+     */
+    private getEventPort(event: MessageEvent) {
```
Comment (jbhanu-thoughtspot) [inline src/embed/base.ts:253]: Why are these changes coming here? Can you rebase your cl?

File: jest.config.sdk.js [hunk 3]
```diff
@@ -6,6 +6,14 @@ module.exports = {
     coverageDirectory: 'coverage/sdk/',
     coverageReporters: ['lcov', 'text', 'cobertura'],
     coveragePathIgnorePatterns: ['/node_modules/', '/test/'],
+    coverageThreshold: {
```
Comment (jbhanu-thoughtspot) [inline jest.config.sdk.js:9]: Let's set all these values into 85. We need higher coverage number

Comment (ashubham) [reply jest.config.sdk.js:9]: Lets also remove `statements` from here.

**Pull request merged**